pyccl/correlations.py

```diff
@@ -220,28 +224,30 @@ def correlation_multipole(cosmo, a, beta, l, s, p_of_k_a=None):
 
     # Convert scalar input into an array
     scalar = False
-    if isinstance(s, (int, float)):
+    if isinstance(r, (int, float)):
         scalar = True
-        s = np.array([s, ])
+        r = np.array([r, ])
 
     # Call 3D correlation function
-    xis, status = lib.correlation_multipole_vec(cosmo, psp, a, beta, l, s,
-                                                len(s), status)
+    xis, status = lib.correlation_multipole_vec(cosmo, psp, a, beta, ell, r,
+                                                len(r), status)
     check(status, cosmo_in)
     if scalar:
         return xis[0]
     return xis
 
 
-def correlation_3dRsd(cosmo, a, s, mu, beta, use_spline=True, p_of_k_a=None):
+@warn_api(pairs=[('s', 'r')])
+def correlation_3dRsd(cosmo, *, a, r, mu, beta,
+                      use_spline=True, p_of_k_a=None):
```

Comment:
Same here, let's swap to `r, a` to follow CCL unwritten rules.
Also, I think `p_of_k_a` should go before `use_spline`, as the latter is just a settings flag for the function.

Reply:
Done

Reply:
Are we not moving `use_spline` after `p_of_k_a`?